app/assets/javascripts/session-timeout.js

```diff
@@ -0,0 +1,251 @@
+;(function (global, document) {
+    'use strict';
+    var $ = global.jQuery;
+    var GOVUK = global.GOVUK || {};
+    var lang = global.jsConfig && global.jsConfig.lang || "en";
+    var strings = {
+        title: {
+            en: "You’re about to be signed out",
+            cy: "Rydych ar fin cael eich allgofnodi"
+        },
+        time: {
+            en: "minutes",
+            cy: "munud"
+        },
+        time_singular: {
+            en: "minute",
+            cy: "munud"
+        },
+        seconds: {
+            en: "seconds",
+            cy: "eiliad"
+        },
+        message: {
+            en: "For security reasons, you will be signed out of this service in",
+            cy: "Am resymau diogelwch, cewch eich allgofnodi o’r gwasanaeth hwn mewn"
+        },
+        keep_alive_button_text: {
+            en: "Stay signed in",
+            cy: "Parhau i fod wedi’ch mewngofnodi"
+        },
+        sign_out_button_text: {
+            en: "Sign out",
+            cy: "Allgofnodi"
+        }
+    };
```

Comment:
if all the messages are in here, do we need to also provide our own messages? Lets just use these!

Reply:
we will need to configure the message for some services like invitations-fe when the user may either be an agent or a client so I have provided the override message. I have removed the other messages that we will not need to change that are defined in the js file.